README.md

```diff
@@ -73,3 +73,22 @@ hardware tests.
 Since this benchmark will cause the kernel image to be an EL2 image, it
 will have an impact on the observed numbers for the other benchmark
 applications as well, since they'll be using an unexpected kernel build.
+
+
+# Adding a new benchmark
+Contributing a new benchmark to seL4bench requires a few steps:
+* Under `apps`, create a directory for your new benchmark and:
```

Comment:
After titles and before lists there should be a blank line in markdown (GitHub will still process it correctly, but other markdown engines don't necessarily)

```suggestion
# Adding a new benchmark

Contributing a new benchmark to seL4bench requires a few steps:

* Under `apps`, create a directory for your new benchmark and:
```